src/borg/helpers.py

```diff
@@ -2218,3 +2218,12 @@ def json_dump(obj):
 
 def json_print(obj):
     print(json_dump(obj))
+
+def secure_erase(path):
+    """Attempt to securely erase a file by writing random data over a file before deleting it."""
+    with open(path, 'r+b') as fd:
+        length = os.stat(fd.fileno()).st_size
+        fd.write(os.urandom(length))
+        fd.flush()
+        os.fsync(fd.fileno())
+    os.unlink(path)
```

Comment:
this file ends with no linefeed, pep8 checkers (and even github code review) complains about this, so please add one (and maybe configure your editor to behave that way).